### PR TITLE
benchmarking with criterion

### PR DESCRIPTION
**Pull request by pacman82**
add benchmarking @BY-stehessel

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -96,6 +80,33 @@ As this tool does not provide any schema validation, it is important to note tha
...
+```bash
+git clone https://github.com/blue-yonder/di-csv2xml.git
+cd di-csv2xml
+cargo build --release
+```
+
+You will find the executable in the `.\target\release\` directory.
+
+You can install cargo from [here](https://rustup.rs) if it is not installed.
+
```
Comment (stephan-hesselmann-by) [inline Readme.md]: Can you rebase this to master to include the installation changes README?

File: Readme.md [hunk 2]
```diff
@@ -96,6 +80,33 @@ As this tool does not provide any schema validation, it is important to note tha
...
+Benchmarks
+----------
+
+This tool uses [criterion](https://github.com/bheisler/criterion.rs) for benchmarking. You can execute the benchmarks with:
+
+```shell
+cargo bench
+```
```
Comment (stephan-hesselmann-by) [inline Readme.md:116]: Execute in which directory? I assume the sources have to be compiled first.

**Review by stephan-hesselmann-by — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (pacman82) [reply Readme.md]: my bad. Done

File: Readme.md (hunk 2 above)
Comment (pacman82) [reply Readme.md:116]: in the repositories direcotry. `cargo` will compile them automatically if they have changed. In release mode.

**Pull request merged**